.bumpversion.cfg

```diff
@@ -0,0 +1,25 @@
+[bumpversion]
+current_version = 5.0.0-dev
+tag = False
+commit = True
+message = EVMC {new_version}
+	
+	Bump version: {current_version} → {new_version}
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?P<prerel>-dev)?
+serialize = 
+	{major}.{minor}.{patch}{prerel}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:prerel]
+optional_value = rel
+values = 
+	-dev
+	rel
+
+[bumpversion:file:CMakeLists.txt]
+
```

Comment:
Will this replace the one in cmakelists?

Reply:
Yes. It will replace "current_version" with "new_version" using matching described in the [bumpversion] section. For evmc.h I have to add different parsing rules to match only major version number and not to replace all `5` digits in the file.

So this should sync EVMC_ABI_VERSION with the semantic version in CMakeLists.txt. We don't use the semantic version anywhere else.